c++/test/TestRleEncoder.cc

```diff
@@ -280,6 +280,33 @@ namespace orc {
     decodeAndVerify(RleVersion_2, memStream, data, numValues, nullptr, isSigned);
   }
 
+  // This case is used to testing encoding large negative integer.
```

Comment:
Is there a case that this would be an issue for positive numbers as well?

Can we also add some checks for the expected encoded values?

Reply:
I have test large positive number, but found it will not use PatchedBase encoding, so it's OK.